.github/workflows/build-push-deploy.yml

```diff
@@ -11,7 +11,8 @@ jobs:
     name: Build Docker Image
     steps:
     - uses: actions/checkout@v3
-
+      with:
+        fetch-depth: 0
```

Comment:
So, it means "fetch all history for all branches and tags"

Reply:
yup